gapic/src/main/com/google/gapid/perfetto/canvas/Panel.java

```diff
@@ -197,6 +203,12 @@ public boolean click() {
           boolean r1 = Hover.this.click(), r2 = onClick.getAsBoolean();
           return r1 || r2;
         }
+
+        @Override
+        public boolean rightClick() {
+          boolean r1 = Hover.this.rightClick(), r2 = onClick.getAsBoolean();
+          return r1 || r2;
```

Comment:
Would suggest simply return Hover.this.rightClick() as well for the same reason. This function withClick() was designed as a helper function, so that we can add new actions to left-click with a lightweight lambda.